source/help/messaging/attaching-files.rst

```diff
@@ -0,0 +1,74 @@
+Attaching Files
+===============
+
+--------------
+
+Attachment Methods
+------------------
+Attach a file by drag and drop or by clicking the attachment icon in the
+message input box.
+
+Drag and Drop
+~~~~~~~~~~~~~
+Upload a file or selection of files by dragging the files from your
+computer into the right-hand sidebar or center pane. Dragging and
+dropping attaches the files to the message input box, then you can
```

Comment:
Feels like a run-on sentence to me. Hard to-read, according to Hemmingway app. Propose: "Dragging and dropping attaches the files to the message input box. Then, you can..."